routes/index.js

```diff
@@ -6,13 +6,13 @@ const {getAllItems, getItem, addItem} = require('../database');
 router.get( '/', function( req, res, next ) {
   getAllItems()
     .then( bookstore => {
-      console.log(bookstore);
       res.render( 'index', { bookstore, title: 'Bookstore App' })
     })
 })
 
 router.post( '/', function( req, res, next ) {
-  const { item } = req.body
+  const item  = req.body
+  console.log(item)
```

Comment:
I see your doing ES6 arrow function's and ES5 fuction:

```() => () VS function() {}```

Just try when you can to use one or the other.